src/trans.c

```diff
@@ -1500,16 +1500,24 @@ Obj FuncON_KERNEL_ANTI_ACTION (Obj self, Obj ker, Obj f, Obj n) {
   UInt    deg, i, j, rank, len;
   Obj     out;
 
-  if (INT_INTOBJ(ELM_LIST(ker, LEN_LIST(ker))) == 0) {
+  assert(IS_LIST(ker));
+  assert(IS_INTOBJ(n));
```

Comment:
So if this is called with bad arguments, GAP crashes, no graceful exit then. Of course the only caller (in the GAP library, at least) of this internal function, namely `OnKernelAntiAction`, does ensure this -- in fact it always sets `n=0`. (Well, and now test call it directly, also with valid inputs).

But I wonder now if we really should be so harsh -- this function uses `ErrorQuit` for other errors. Maybe it should use that, or `ErrorMayQuit`, here, too, just as a gesture of friendliness to people who feel the need to directly use this low-level version?

Alas, I am just wondering -- I am also happy to merge this PR as-is!